README.md

```diff
@@ -276,12 +276,12 @@ Now that you have created two tables, you'll see entries for them in the `pointc
 > 
 >     1     
  
-**PC_Envelope(p pcpatch)** returns **bytea**
+**PC_Envelope_AsBinary(p pcpatch)** returns **bytea**
```

Comment:
I'd remove this function completely. If I want to get an OGC WKB representation of the envelope polygon I can just do:

```SQL
select ST_AsBinary(PC_Envelope(patch));
```

We don't need PointCloud to provide a specific function for that.